source/features/transactions/components/TransactionListRow.tsx

```diff
@@ -0,0 +1,93 @@
+import { observer } from 'mobx-react-lite';
+import moment from 'moment';
+import React from 'react';
+import styles from './TransactionListRow.scss';
+
+const ArrowNext = require('../../../static/assets/images/arrow-next.svg');
+const SEVEN_DAYS = 7 * 24 * 3600000;
+
+const shrinkAddress = (address: string) => {
+  const N = address.length;
```

Comment:
Can you rename this variable to something else like: addressLength or lengthOfAddress . We don't usually use 1 character naming for variables. Or maybe just remove it and use directly address.length in the if condition bellow.